packages/fx-core/templates/core/v3Migration/csharp.app.local.yml

```diff
@@ -78,7 +78,7 @@ configureApp:
 {{#if activePlugins.fx-resource-aad-app-for-teams}}
   - uses: aadApp/update # Apply the AAD manifest to an existing AAD app. Will use the object id in manifest file to determine which AAD app to update.
     with:
-      manifestTemplatePath: ./aad.manifest.template.json # Relative path to teamsfx folder. Environment variables in manifest will be replaced before apply to AAD app
+      manifestPath: ./aad.manifest.template.json # Relative path to teamsfx folder. Environment variables in manifest will be replaced before apply to AAD app
```

Comment:
shall we change the comment "to teamsfx folder" here to "this file"?
plz review.

Reply:
let's do it (also for other similar comments in the C#/js/ts yml file)